ci/lava/conftest.py

```diff
@@ -0,0 +1,193 @@
+    parser.addoption("--debug_output", action="store_true")
+    parser.addoption("--device", action="store", default="none")
+    parser.addoption("--dut", action="store", default="auto")
+    parser.addoption(
+        "--dut_download_dir", action="store", default="/tmp/pytest"
+    )
+    parser.addoption("--dut_tutorials_dir", action="store", default="/scratch")
+    parser.addoption(
+        "--host_download_dir", action="store", default="/tmp/pytest"
+    )
+    parser.addoption(
+        "--host_tutorials_dir", action="store", default="/tmp/tutorials"
+    )
+    parser.addoption("--venv", action="store", default="/tmp/venv")
+
+
+@pytest.fixture
+def debug_output(request):
+    """Fixture for --debug_output."""
+    return request.config.getoption("--debug_output")
+
+
+@pytest.fixture
+def device(request):
+    """Fixture for --device."""
+    return request.config.getoption("--device")
+
+
+@pytest.fixture
+def dut(request):
+    """Fixture for --dut."""
+    return request.config.getoption("--dut")
+
+
+@pytest.fixture
+def dut_download_dir(request):
+    """Fixture for --dut_download_dir."""
+    return request.config.getoption("--dut_download_dir")
+
+
+@pytest.fixture
+def dut_tutorials_dir(request):
+    """Fixture for --dut_tutorials_dir."""
+    return request.config.getoption("--dut_tutorials_dir")
+
+
+@pytest.fixture
+def host_download_dir(request):
+    """Fixture for --host_download_dir."""
+    return request.config.getoption("--host_download_dir")
+
+
+@pytest.fixture
+def host_tutorials_dir(request):
+    """Fixture for --host_tutorials_dir."""
+    return request.config.getoption("--host_tutorials_dir")
+
+
+@pytest.fixture
+def venv(request):
+    """Fixture for --venv."""
+    return request.config.getoption("--venv")
+
+
+class Execute_Helper:
+    """Class to provide a wrapper for executing commands as a subprocess."""
+
+    @staticmethod
+    def _print(data):
+        print(data)
+
+    @staticmethod
+    def _execute_command(command):
+        """Execute the provided command list.
+
+        Executes the command and returns the error code, stdout and stderr.
+        """
+        Execute_Helper._print("execute_command start:")
+        Execute_Helper._print("command:")
+        Execute_Helper._print(command)
+        p = subprocess.Popen(
+            command,
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            bufsize=-1,
+            universal_newlines=True,
+        )
+        output, error = p.communicate()
+        Execute_Helper._print("output:")
+        Execute_Helper._print(output)
+        Execute_Helper._print("error:")
+        Execute_Helper._print(error)
+        Execute_Helper._print("returnCode:")
+        Execute_Helper._print(p.returncode)
+        Execute_Helper._print("execute_command done.")
+
+        return p.returncode, output, error
+
+    @staticmethod
+    def _send_mbl_cli_command(command, addr):
```

Comment:
I'm not sure what the public interface of this class is. looking at the test examples it looks like this `_send_mbl_cli_command` method. If that is the case we should remove the leading underscore, as having it there implies this method is an implementation detail.

Reply:
changed.